vpr/src/place/place.cpp

```diff
@@ -1462,7 +1462,7 @@ static e_move_result try_swap(const t_annealing_state* state,
         rlim = state->rlim;
     }
 
-    e_create_move create_move_outcome;
+    e_create_move create_move_outcome = e_create_move::ABORT;
 
```

Comment:
I think we should move the #ifndef NO_GRAPHICS here and the #endif after "} else" and before the "if" in line 1472 so that the manual_move_enabled will only be checked if the graphics are enabled (NO_GRAPHICS isn't defined). Then, create_move_outcome will be initialized in all the cases.

Reply:
@hzeller : any thoughts on Mohamed's comment?  I'm OK with merging as is, but if you and @MohamedElgammal have a better fix please make the change now so we fix this one way or the other.